# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/irda-utils/gcc-__func__.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- irda-utils-0.9.16/tekram/irkbd.c	2004-10-27 19:11:15.553129704 +0000
+++ irda-utils-0.9.16-p/tekram/irkbd.c	2004-10-27 19:11:44.404743592 +0000
@@ -298,7 +298,7 @@
 	case IRKBD_RSP_KBDSPEEDOK:
 		break;
 	default:
-		printf(__FUNCTION__ "(), unknown response %02x\n", rsp);
+		printf(__func__, "(), unknown response %02x\n", rsp);
 		break;
 	}
 }
